messages/common.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
+	flags: {
+		verboseSummary: 'emit additional command output to stdout'
```

Comment:
`SfCommand` has no built-in `--verbose` flag. So this text is directly copied from the summary that `SfdxCommand` provides for its built-in `--verbose` flag.